### PR TITLE
Restaurant Page

### PR DESCRIPTION
**Pull request by imran5644**
This project is A landing page for a restaurant where you can visit.
1)There are three pages 
I) Home page
II)Contact page
III) Menu Page
And You can see the menu with a price.
I built this project with
Webpack
VS Code
JavaScript**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,79 @@
...
+- Open Terminal
+- Change directory to folder to download repository
+- Use `cd <file-path>`
+- Run `git clone https://github.com/imran5644/RESTAURANT_PAGE.git`
+
+
+
+To make any change,
+
+- Open related file by using text editor.
```
Comment (whiz25) [inline README.md]: - What related file? Be as explicit as possible

File: README.md [hunk 2]
```diff
@@ -1 +1,79 @@
...
+
+- Install [git](https://git-scm.com/downloads)
+- Open Terminal
+- Change directory to folder to download repository
+- Use `cd <file-path>`
+- Run `git clone https://github.com/imran5644/RESTAURANT_PAGE.git`
+
+
+
+To make any change,
```
Comment (whiz25) [inline README.md]: - This instruction is related to contributing and should be moved there.

File: README.md [hunk 3]
```diff
@@ -1 +1,79 @@
...
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Text Editor (VSCode is suggested.)
+
```
Comment (whiz25) [inline README.md:29]: - One also need `node` for installing packages. Add that as a prerequisite.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Amazing work. I like the design. The color harmony is impressive. You also managed to make use of ES6 syntax and modules as requested. Well done!
Let's make some changes to the readme file to make it more professional.
- [ ] check my comments for suggestions.

### Martin Afani
#### Slack: whiz25

**Review by wathigo — APPROVED** [review]
### APPROVED!
Great job implementing the suggested changes.

**You may proceed to the next project.**

**Happy weekend :student: 
Simon Wathigo**

**Pull request merged**